src/System.Net.WebSockets.Client/tests/ClientWebSocketUnitTest.cs

```diff
@@ -1,39 +1,45 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.Net.WebSockets;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Text;
 
 using Xunit;
+using Xunit.Abstractions;
 
-namespace System.Net.WebSockets.Client.Test
+namespace System.Net.WebSockets.Client.Tests
 {
     /// <summary>
     /// ClientWebSocket unit tests that do not require a remote server.
     /// </summary>
-    public class ClientWebSocketTest
+    public class ClientWebSocketUnitTest
     {
-        internal static class AssertExtensions
+        private readonly ITestOutputHelper _output;
+
+        public ClientWebSocketUnitTest(ITestOutputHelper output)
         {
-            public static void Throws<T>(Action action, string message)
-                where T : Exception
-            {
-                Assert.Equal(Assert.Throws<T>(action).Message, message);
-            }
+            _output = output;
         }
 
         [Fact]
-        public void ClientWebSocket_Ctor()
+        public void Ctor_Success()
         {
+            if (WebSocketHelper.ShouldSkipTestOSNotSupported(_output))
+            {
+                return;
+            }
+
```

Comment:
I will work on this extensions attributes as part of the next WebSockets PR I'll be doing.